src/Microsoft.DotNet.CodeAnalysis.net45/Microsoft.DotNet.CodeAnalysis.net45.csproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
```

Comment:
Setting CopyNuGetImplementations=true is fine but we aren't currently using them for anything because we aren't putting them into the BuildTools package so it isn't actually necessary.